PocketKit/Sources/PocketKit/Article/ReadableViewModel/RecommendableItemViewModel.swift

```diff
@@ -406,7 +406,7 @@ extension RecommendableItemViewModel {
 
 // MARK: instantiation from url
 extension RecommendableItemViewModel {
-    static func fromURL(_ urlString: String) -> RecommendableItemViewModel? {
+    static func fromURL(_ urlString: String, readableSource: ReadableSource) -> RecommendableItemViewModel? {
```

Comment:
injecting `readableSource` here as this method is now used by callers with different `ReadableSource` values